routes/error-tracker.js

```diff
@@ -225,25 +237,25 @@ function getHandler(req, res) {
     },
     severity: severity,
   };
-  const entry = log.entry(metaData, event);
-  log.write(entry, function(err) {
-    if (err) {
-      res.status(statusCodes.INTERNAL_SERVER_ERROR);
-      res.send({error: 'Cannot write to Google Cloud Logging'});
-      res.end();
-      winston.error(appEngineProjectId, 'Cannot write to Google Cloud Logging: '
-        + url.parse(req.url, true).query['v'], err);
-    }
+  return unminify.unminify(exception).then(function(unminifiedException) {
+    exception = params.m + '\n' + unminifiedException;
+    const entry = log.entry(metaData, event);
```

Comment:
We need to set `event.message`, simply updating `exception` variable will not mutate the object. There should be a test for this.